gammapy/cube/make.py

```diff
@@ -257,3 +260,125 @@ def _check_selection(selection):
             raise ValueError("Selection not available: {!r}".format(name))
 
     return selection
+
+class IrfMapMaker:
+    """Make IRF maps from IACT observations.
+
+    Parameters
+    ----------
+    geom : `~gammapy.maps.WcsGeom`
+        Reference map geometry with true energy axis
+    offset_max : `~astropy.coordinates.Angle`
+        Maximum offset angle
+    rad_axis : `~gammapy.maps.MapAxis`
```

Comment:
Currently this is the only case in Gammapy where we accept `MapAxis` as argument to a function / class (except for `MapGeom.__init__()`). I'd slightly prefer to just pass a vector of `rad` and `migra` values and create the `MapAxis` inside. But I don't have a strong opinion here...

Reply:
OK. Right, I change accordingly.